manage-applications.md

```diff
@@ -195,12 +199,6 @@ For security reasons, the target location of the files and directories listed in
 * `/data/app/<apk-package-name>`
 * `/data/data/<apk-package-name>`
 
-The `image` attribute defines which image the application is based on. If left empty, your application will be based on the default image. See [Image Management](https://discourse.ubuntu.com/t/managing-images/17758) for more details on this. Available images on your installation can be listed with the following command:
```

Comment:
This part seemed unrelated to me, so I moved it to a separate section (higher up). Does that make sense?